development-vm/README.md

```diff
@@ -1,276 +1,226 @@
-# Developing on GOV.UK Virtual Machines
+# Getting Started
 
-Welcome to GOV.UK. The following instructions show you how to get your
-development environment running.
+Welcome to GOV.UK.
 
-## 0. Context
+Our development environment is an Ubuntu virtual machine (VM). We aim to achieve [dev-prod parity](http://www.12factor.net/dev-prod-parity).
 
-Our development environment is an Ubuntu virtual machine with a view
-to achieving [dev-prod parity][1]. By default, the steps below will
-set you up with a [VirtualBox][2] VM, managed and configured by
-[Vagrant][3]. If you feel strongly about using another piece of
-software (such as VMWare) for your development VM, you may find
-instructions for doing so [on the wiki][4].
+Follow the steps on this page to get your GDS development environment running with a [VirtualBox](https://www.virtualbox.org/) VM, managed and configured by [Vagrant](http://vagrantup.com/).
 
-Either way, you will need virtualisation enabled in your BIOS, otherwise it
-won't work. This tends to be enabled by default on Macs, but is worth
-checking for other manufacturers.
+> You'll need to use a Mac to follow this guide.
 
-[1]: http://www.12factor.net/dev-prod-parity
-[2]: https://www.virtualbox.org/
-[3]: http://vagrantup.com/
-[4]: https://github.com/alphagov/wiki/wiki
+It will take you roughly one day to do everything in this guide from start to finish. There are lots of things to download, and loads of installers need to do their thing.
 
-## 1. Prerequisites and assumptions
+> You'll need up to 50GB of free space on your hard-drive to run the whole of GOV.UK.
 
-  * You have [OSX GCC tools installed](https://github.com/kennethreitz/osx-gcc-installer) (or via XCode)
-  * You have [VirtualBox](https://www.virtualbox.org/) installed
-  * You have [Vagrant](https://www.vagrantup.com/downloads.html) installed
-  * you have vagrant-dns plugin installed ` vagrant plugin install vagrant-dns `
-  * You have an account on [GitHub.com](https://github.com)
-  * You have some other repositories from GOV.UK checked out to work on -
-    these should be located alongside the `govuk-puppet` repository we're in
-    now (e.g `~/govuk/govuk-puppet:~/govuk/frontend`)
+**Developing outside the VM**
 
-### A note on Boxen
+You don't have to develop on the VM, but we strongly recommend it. If you have problems with the development VM you can always ask for help in the #govuk-developers Slack channel.
 
-If you have a GDS issued laptop and you want to automate most of this,
-consider building your laptop using
-[GDS Boxen](https://github.com/alphagov/gds-boxen). Specifically
-you'll want these things:
+**Where you should run commands**
```

Comment:
I tried this and it didn't work. As in, it broke the entire documentation site somehow. Eventually I got bored of trying to fix it and gave up.

Suspect that our navigation builder requires a 'proper' hierarchy of headers. Since this content exists before the first `h2`, it won't them be `h3`s.